src/samples/http/server-sse.c

```diff
@@ -72,14 +72,28 @@ static struct sol_ptr_vector responses = SOL_PTR_VECTOR_INIT;
 static int port = 8080;
 static bool should_quit = false;
 
+static struct sol_blob sse_prefix = {
+    .type = &SOL_BLOB_TYPE_NO_FREE,
+    .parent = NULL,
+    .mem = (void *)"data: ",
+    .size = sizeof("data: ") - 1,
+    .refcnt = 1
+};
+
+static struct sol_blob sse_suffix = {
+    .type = &SOL_BLOB_TYPE_NO_FREE,
+    .parent = NULL,
+    .mem = (void *)"\n\n",
+    .size = sizeof("\n\n") - 1,
+    .refcnt = 1
+};
+
```

Comment:
said above, we should create a `sol_http_progressive_response_send_sse()` or `sse_data()`, where it will take a blob with the actual data, will queue the prefix and suffix for you.